plugins/backstage-plugin-flux/src/plugin.ts

```diff
@@ -163,11 +163,24 @@ export const FluxRuntimeCard = weaveworksFluxPlugin.provide(
  * Page used to show Flux Controllers / Deployments in Flux Runtime
  * @public
  */
-export const TechRadarPage = weaveworksFluxPlugin.provide(
+export const FluxRuntimePage = weaveworksFluxPlugin.provide(
   createRoutableExtension({
     name: 'TechRadarPage',
```

Comment:
```suggestion
    name: 'FluxRuntimePage',
```

Not sure where/if this gets used